<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="job_settings_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
         <item>
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,1">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Date shipped:</string>
             </property>
             <property name="buddy">
              <cstring>dateEdit_shipped</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QDateEdit" name="dateEdit_expected">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_type">
             <item>
              <property name="text">
               <string>Planning</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Quoting</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Quoted</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Template</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Workspace</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Archive</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Date expected:</string>
             </property>
             <property name="buddy">
              <cstring>dateEdit_expected</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_order_number">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="maximum">
                <double>1000000000000000000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_get_order_number">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Get latest order number</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Order number:</string>
             </property>
             <property name="buddy">
              <cstring>doubleSpinBox_order_number</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QDateEdit" name="dateEdit_shipped">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Job status:</string>
             </property>
             <property name="buddy">
              <cstring>comboBox_type</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Ship to:</string>
               </property>
               <property name="buddy">
                <cstring>textEdit_ship_to</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_reload_job">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Reload current job

Ctrl + R</string>
               </property>
               <property name="styleSheet">
                <string notr="true">
QPushButton#pushButton_reload_job:flat {
    background-color: transparent;
    border: 1px solid transparent;
    border-radius: 12px;
    padding: 5px 10px;
    color: #EAE9FC;
}

QPushButton#pushButton_reload_job:flat:pressed {
    background-color: #3daee9;
    color: #171717;
}

QPushButton#pushButton_reload_job:flat:hover {
    background-color: rgba(76, 76, 76, 0.6);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>svg/browser_refresh.svg</normalon>
                </iconset>
               </property>
               <property name="shortcut">
                <string>Ctrl+R</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit_ship_to">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Information about where this job is shipped to...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="groups_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1042</width>
            <height>202</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="groups_layout"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="add_group_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add Group</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
